res/themes/Dark Blue/client/client.css

```
@@ -0,0 +1,1617 @@
+    background-color: #1F2937;
+    color: #E5E7EB;
+    min-width: 80px;
+}
+
+QTabWidget > QTabBar::tab:hover
+{
+    color: #F9FAFB;
+    background-color: #374151;
+}
+
+QTabWidget > QTabBar::tab::selected
```

Biome (2.1.2)

[error] Unknown type selector is not allowed.

See MDN web docs for more details.
Consider replacing the unknown type selector with valid one.

(lint/correctness/noUnknownTypeSelector)